frontend/src/app/projects/project-detail/project-users/project-user-settings.component.html

```diff
@@ -37,11 +36,13 @@ <h2>Project members</h2>
       />
       <mat-icon matSuffix>search</mat-icon>
     </mat-form-field>
-    <div class="scrollable">
+    <div class="verflow-y-scroll max-h-[50vh]">
```

Comment:
The overflow scrolling doesn't work in the mobile view. 

![image](https://github.com/DSD-DBS/capella-collab-manager/assets/23395732/f440a5f3-116b-43ab-a861-6abd5d26bbaf)

Reply:
The overflow also didn't work in the normal view. But this was just a typo, i.e., there was just an `o` missing in front